books/free-programming-books-langs.md

```diff
@@ -373,6 +373,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Bash tutorial](https://web.archive.org/web/20180328183806/http://gdrcorelec.ups-tlse.fr/files/bash.pdf) - Anthony Scemama (PDF)
 * [BashGuide](http://mywiki.wooledge.org/BashGuide) - Maarten Billemont (HTML) [(PDF)](http://s.ntnu.no/bashguide.pdf)
 * [Conquering the Command Line](http://conqueringthecommandline.com/book/frontmatter) - Mark Bates (HTML)
+* [Essentials - Conquer the Command Line](https://magpi.raspberrypi.org/books/command-line-second-edition/pdf/download) - Richard Smedley (PDF)
```

Comment:
⚠️ Use the authentic book title is a must. See book cover.

Reply:
Are you referring to changing it to `Conquer the Command Line`

Reply:
You are right

Reply:
https://github.com/EbookFoundation/free-programming-books/pull/5787/commits/3422272fa024170f4db0d6a115e8c3775d35d95b